deployment/environment/memory/chain.go

```diff
@@ -16,30 +15,13 @@ import (
 	chainsel "github.com/smartcontractkit/chain-selectors"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
-	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/testcontext"
 )
 
 type EVMChain struct {
 	Backend     *simulated.Backend
 	DeployerKey *bind.TransactOpts
 }
 
-// CCIP relies on block timestamps, but SimulatedBackend uses by default clock starting from 1970-01-01
-// This trick is used to move the clock closer to the current time. We set first block to be X hours ago.
-// Tests create plenty of transactions so this number can't be too low, every new block mined will tick the clock,
-// if you mine more than "X hours" transactions, SimulatedBackend will panic because generated timestamps will be in the future.
```

Comment:
It seems the new simulated backend doesn't care about timestamps in the future so we don't need to do this anymore